astropy/units/core.py

```diff
@@ -70,6 +70,56 @@
 }
 
 
+class _UnitPrefix(NamedTuple):
+    symbol: tuple[str]  # wrapped in a tuple for backwards compatibility
```

Comment:
I could investigate if `str` could be used instead of `tuple[str]` if the `units` maintainers also think it would be a good idea, but I wanted to open this pull request before doing that to demonstrate that using tuples instead of lists is backwards compatible.

Reply:
I could see the advantage of keeping a list if eventually we'd want to have both "u" and "μ" as prefixes - the latter is the correct one after all (just difficult to type on most keyboards... though compose-m-u works for me in emacs/terminal).

Reply:
I wonder about symbol - should it perhaps be "short" and "long"? For the binary prefixes, it is not really a "symbol". But I'm also fine to keep as is.

Reply:
> I could see the advantage of keeping a list if eventually we'd want to have both "u" and "μ" as prefixes

I'll let `symbol` be a tuple then.

> I wonder about symbol - should it perhaps be "short" and "long"? For the binary prefixes, it is not really a "symbol". But I'm also fine to keep as is.

I used the same vocabulary as the official [SI Brochure](https://www.bipm.org/documents/20126/41483022/SI-Brochure-9-EN.pdf) (see e.g. Table 7).